resources/routing_rule.rb

```diff
@@ -0,0 +1,48 @@
+property :description, ::String, default: ''.freeze
+property :mode, ::String, equal_to: %w[BLOCK ALLOW], default: 'BLOCK'.freeze
+property :matchers, Array, default: lazy { [] }, coerce: proc { |r| r.sort }
+property :api_client, ::Nexus3::Api, identity: true, default: lazy { ::Nexus3::Api.default(node) }
+
+load_current_value do |desired|
+  begin
+    config = api_client.routing_rule(desired.name)
```

Comment:
Use the suggested `rule_name` property instead

Reply:
Done